FlorenceBML/BMLacq811.xml

```diff
@@ -7,8 +7,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
             <titleStmt>
                 <title/>
                 <editor key="AB" role="generalEditor"/>
+                <editor key="CH"></editor>
                 <funder>Akademie der Wissenschaften in Hamburg</funder>
             </titleStmt>
+            <editionStmt>
```

Comment:
I guess we have no images so probably this passage should go away, will check tomorrow